augur/clades.py

```diff
@@ -102,12 +102,9 @@ def assign_clades(clade_designations, all_muts, tree, ref=None):
     '''
 
     clade_membership = {}
+    basal_clade_nodes = {}
     parents = get_parent_name_by_child_name_for_tree(tree)
 
-    # first pass to set all nodes to unassigned as precaution to ensure attribute is set
```

Comment:
In the function's docstring and in block comments throughout there are several references to the number of passes through the tree and to the former hardcoded attribute names (`clade_membership` and `clade_annotation`). We should update or remove these references, to avoid future confusion.

Reply:
Done 👍